mehl.cloud.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Csl. Thomas Lüchow
+status: Teacher and Coach for Adults
+domains: 
+  - mehl.cloud
+server:
+  - type: IMAP
+    socket: TLS
+    hostname: mehl.cloud
+    port: 993
+    username_pattern: EMAIL
+  - type: SMTP
+    socket: STARTTLS
+    hostname: mehl.cloud
+    port: 587
+    username_pattern: EMAIL
+before_login_hint: Wellcome to mehl.cloud
+after_login_hint: Now you are loged in
```

Comment:
```suggestion
```
I would leave out those two lines, they are only meant for giving users tips on how to configure the server. With chatmail, this is not needed.